resources/prometheus/remote-write.yaml

```diff
@@ -33,20 +33,126 @@ writeRelabelConfigs:
       - __name__
     targetLabel: __tmp_keep
 
+  # WHAT: Reports the current amount of samples that have tried to be resent to observatorium
+  - action: replace
+    regex: container_cpu_usage_seconds_total*$
```

Comment:
Not sure why there are 2 entries for the same metrics?

Reply:
@shivamG640  Will have a look on that